sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeUtils.scala

```diff
@@ -81,6 +81,12 @@ object DateTimeUtils {
     }
   }
 
+  private val threadLocalUtcGregorianCalendar = new ThreadLocal[GregorianCalendar] {
+    override def initialValue(): GregorianCalendar = {
+      new GregorianCalendar(TimeZoneUTC)
```

Comment:
does timezone matter here?

Reply:
Based on the implementation (https://github.com/JetBrains/jdk8u_jdk/blob/master/src/share/classes/java/util/GregorianCalendar.java#L819), I don't think timezone would make a difference here.